cookbooks/fb_ssh/libraries/default.rb

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2019-present, Vicarious, Inc.
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module FB
+  class SSH
+    DESTDIR = {
```

Comment:
I think these should probably be configurable (not everybody will have/want the same paths).

Reply:
I disagree...  one of the things we've almost never done in fb cookbooks is make configuration paths configurable....